### DISCUSSION TITLE
Capacitor finding different NodeJS version from what 'node --version' returns

### DISCUSSION THREAD

**Discussion by lcrumling** [ionic-team/capacitor#5981] [Q&A]
I'm trying to migrate an existing app from Cordova to Capacitor, and I got through all the migration steps and updating all the dependencies, but now Capacitor is throwing a weirdly simple error about my NodeJS version not being greater than 12.4.0, even though `node --version` is returning `v14.20.1` which should satisfy the requirement. I was able to successfully run npx cap commands before I updated the dependencies, but I'm not sure how to find out what is causing this issue since everything I check is returning what I expect (node --version, which node), except for npx cap commands.

I did see a similar issue posted about this, but in that case, the user was on 'v16.0.0-head' and the -head was throwing off the version check, which is not the case here (also that issue says it's now fixed).

I tested creating a new Capacitor app in a separate folder, and that went fine - I was able to run `npx cap -V` successfully without the NodeJS version error.

How can I tell where Capacitor is getting this node version from, and correct it?
Any help is appreciated, thank you!!

Capacitor Version
`npx cap -V` output 
`[fatal] The Capacitor CLI requires NodeJS >=12.4.0`

Platform(s)
node 14.20.1
MacOs Monterey 12.1, Intel Core i9

Current Behavior

`npx cap -V` and  `npx cap sync
`
return
`[fatal] The Capacitor CLI requires NodeJS >=12.4.0
`

Expected Behavior
Capacitor returns version, syncs

Code Reproduction

Other Technical Details
`npm --version` output:
`8.19.2`

`node --version` output:
`v14.20.1`

`pod --version` output (iOS issues only):
`1.11.3`

Additional Context
<img width="581" alt="Screen Shot 2022-10-08 at 10 36 27 AM" src="https://user-images.githubusercontent.com/44472933/194713292-9018e8a9-3a6e-4b67-b744-720294ad1da0.png">

**Comment by jcesarmobile** [comment]
Looks like the problem is the v in v14.20.1, it shouldn’t have that, how did you install node?

**Reply by lcrumling** [reply]
oh interesting, thank you! I installed it via nvm, and I also installed nodenv and tried installing 14.20.1 again but got the same result.

**Reply by lcrumling** [reply]
Oh and I also get the same output (with the ‘v’) when running `node —version` in the empty app.

**Reply by jcesarmobile** [reply]
yeah, sorry, just tested on my computer and it also has the v on the version name.

Can you do this quick test?
Create a file named `test.js` and put this line of code inside `console.log(process.version);`.
Then run `node test.js`.

`process.version` is what Capacitor CLI uses to check your node version.
